samples/hello_fpga/hello_fpga_n3000.c

```diff
@@ -0,0 +1,518 @@
+// Copyright(c) 2017-2020, Intel Corporation
```

Comment:
Instead of adding this new program, why not just add a command-line option to the existing hello_fpga.c so that the different AFUID can be passed in.  You've replicated the entire program for the sake of one #define.

Reply:
yes, i will add a new argument to support the N3000. for example, add "c" argument.

[figo@localhost hello_fpga]$ sudo ./hello_fpga -c
Running Test
Running on bus 0x08.
AFU NLB0 found @ 28000
Done Running Test